xml/System.Web.UI.WebControls/LinqDataSourceView.xml

```diff
@@ -1155,9 +1212,17 @@
         <ReturnType>System.Web.UI.WebControls.ParameterCollection</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a collection of parameters whose values are used to create the GroupBy clause during the select operation.</summary>
```

Comment:
I think "GroupBy" needs code brackets to prevent translation. This same problem is in the `<value>` node below.

Reply:
This is ECMAXML, so we shouldn't use Markdown concepts. I'll leave as-is for now.